tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/Lambda.cs

```diff
@@ -60,7 +60,7 @@ internal static CallTargetState OnMethodBegin<TTarget>(TTarget instance, object
                 Console.WriteLine(ex);
             }
 
-            return CallTargetState.GetDefault();
+            return new CallTargetState(CreateDummyScope(Tracer.Instance));
```

Comment:
I know I introduced the term 'Dummy', but maybe we should use something like 'FacadeScope' or 'PlaceholderScope' instead since they are more neutral terms.